core/src/test/java/com/linecorp/armeria/internal/server/annotation/AnnotatedServiceTypeUtilTest.java

```diff
@@ -153,4 +161,13 @@ public static StringConstructor valueOf(String str) {
             return new StringConstructor(str);
         }
     }
+
+    public enum EnumWithOfCreator {
+        FOO,
+	BAR;
```

Comment:
Could you use whitespaces instead of a tab?

Reply:
oh :blush: My bad, editing on another computer now.

OT: Could Armeria add a .editorconfig (which seems to be the de-facto standard) with the correct indentation settings? Lowers the bar of (correct) contributions quite a bit.

Reply:
> Was editing in Vim... no .editorconfig support by default in that anyway.

🤣 

> OT: Could Armeria add a .editorconfig (which seems to be the de-facto standard) with the correct indentation settings? Lowers the bar of (correct) contributions quite a bit.

That's a good suggestion! Would you mind sending a PR for that? or we could take care of it. 😄

Reply:
Perhaps we could copy https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/master/.editorconfig

Reply:
I'm not sure that's my place to say, maybe @trustin, @ikhoon or someone else with more skin than me in the game? (Sorry, don't know who's who in Armeria).

@minwoox I *think* I removed all the tabs now.

Reply:
> I'm not sure that's my place to say, maybe @trustin, @ikhoon or someone else with more skin than me in the game? (Sorry, don't know who's who in Armeria).

Don't worry, we can take care of it. 😄 

> I think I removed all the tabs now.

Thanks! 🙇‍♂️